1906-06-23.xml

```diff
@@ -0,0 +1,1587 @@
+                            depth of water varies from zero to 2 metres on the men's side and to 1)
+                            metres on the ladies' side. Between the two sides a large terrape bas
+                            been erected where on a buffet bas been installed. For those who wish
+                            for nothing more than a "blow" in these warm months, the baths will be
+                            deoided boon,</p>
+                    </div>
+                </div>
+                <div type="section">
+                    <head>PERSONAL AND SOCIAL</head>
+                    <div type="item">
+                        <p>Ahmed Paoba Yehis entertained at a tea party yesterday H.E. the Regent,
+                            Sir R Wingate Pacha, Msz'oum Pacha, Pakhry Pacha, Abani Paobs, Ghalib
+                            Pachs Sardar, Mahmoad Pacha Shukri, and Mastapha Pacha Ibady, Govərnor
+                            of Alexandria.</p>
+                    </div>
+                    <div type="item">
+                        <p>H. E. Hussein Pacha Fakhry, Minister of Pablic Works, will leave for
+                            England next Monday</p>
+                    </div>
+                    <div type="item">
+                        <p>M. de la Boulinière, French Diplomatic Agent and Consal-General in Cairo,
+                            is leaving to-day, via Port Said, for France, where he will spend the
+                            summer months. M. Campon, second secretary at the French Agency, will
+                            act as Consul-General during his at seuce.</p>
+                    </div>
+                    <div type="item">
+                        <p>Baron Max Oppenheim left Cairo to-day for Alexandria, where he will
+                            embark for Europe. Baron Oppenheim will spend the summer at Obergassel,
+                            Germany, returning to Egypt towards the end of October.</p>
+                    </div>
+                    <div type="item">
+                        <p>Dr. Antoine Comanos Parbai will leave for Europe on the 29th inst.</p>
+                    </div>
+                    <div type="item">
+                        <p>Dr. Wigham, of Port Said, has just been in cribed as an Avocat Cour
+                            d'Appel. Dr. Wigbam, who is an English lawyer of some yaars' standing,
+                            purposes, we understand, to practise in the Court of H.B.M.'s Consal
+                            General at Port Said.</p>
+                    </div>
+                    <div type="item">
+                        <p>Said Bey Shukeir, of the Sudan Administration, left Cairo yesterday,
+                            accompanied by bis family, for the Lebanon, where they will spend the
+                            summer season.</p>
+                    </div>
```

Comment:
close the div here--advert that follows is a sibling, not part of the social section